src/components/VideoCards/VideoCards.component.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Card } from '../Card';
+import VideoWrapper from './VideoCards.styled';
+
+function VideoCardList({ videos }) {
+
+  return (
+    <VideoWrapper>
+      {videos.map((video) => (
+        <Card
+          key={video.id}
+          img={video.image}
+          title={video.title}
+          description={video.description}
+        />
```

Comment:
Hey @javierreyees!
You forgot to add a unique key attribute to elements that are part of an array. 👀